guides/doc-Provisioning_Guide/topics/Cloud-Azure.adoc

```diff
@@ -0,0 +1,258 @@
++
+[options="nowrap" subs="+quotes"]
+----
+# hammer compute-profile create --name _compute_profile_name_
+----
+
+. Add Azure details to the compute profile: 
+[options="nowrap" subs="+quotes"]
+----
+# hammer compute-profile values create \
+--compute-profile "_compute_profile_name_" \ <1>
+--compute-resource _azure_cr_name_ \         <2>
+--organization "_Your_Organization_" \       <3>
+--location "_Your_Location_" \               <4>
+--compute-attributes="resource_group=_resource_group_,vm_size=_Standard_B1s_,username=_azure_user_,password=_azure_password_,platform=Linux" \ <5>
+--interface="compute_public_ip=Dynamic,compute_network=mysubnetID,compute_private_ip=false" <6>
+----
+<1> With the `--compute-profile` option, enter the name of a compute profile to configure.
+<2> With the `--compute-resource` option, enter a name of the Azure Compute resource that you want to configure a compute profile for.
+<3> With the `--organization` option, enter an organization to use.
+<4> With the `--location` option, enter a location to use.
+<5> With the `--compute-attributes` option, specify the following details:
+* With the `resource_group` setting, enter the resource group to provision to.
+* With the `vm_size` setting, enter the size of the VM to provision, for example `Standard_B1s`.
+* With the `username` setting, enter the SSH user name for image access. Note that the username that you enter for compute profile must be the same that you use when creating an image.
+* With the `password` setting, enter the password to authenticate with.
+* With the `platform` setting, enter `Linux`.
+<6> With the `--interface` option, specify the following details:
+* With the `compute_public_ip` setting, enter the public IP address type. Must be either `None`, `Static`, or `Dynamic`.
+* With the `compute_network` setting, enter the ID of an available Azure Subnet to use for provisioning.
+* With the `compute_private_ip` setting, specify whether to use a private IP address. Must be `true` or `false`.
+
+[[Provisioning_Virtual_Machines_in_Azure-Creating_Image_Based_Hosts_on_Azure]]
+=== Creating Image-Based Hosts on Azure
+
+The Azure provisioning process creates hosts from existing images on Azure.
+
+.Procedure
+
+. In the {Project} web UI, navigate to *Hosts* > *Create host*.
+. In the *Name* field, enter a name for the host.
+. Click the *Organization* and *Location* tabs to ensure that the provisioning context is automatically set to the current context.
+. Optional: From the *Host Group* list, you can select a host group to populate most of the new host's fields.
+. From the *Deploy on* list, select an Azure connection.
+. From the *Compute Profile* list, select a profile to use to automatically populate virtual machine-based settings.
+. Click the *Virtual Machine* tab and verify that the fields are populated with the correct values from the selected compute profile. 
+. Click the *Interface* tab and click *Edit* on the host's interface.
```

Comment:
If Static Private IP checkbox had been checked, user has to consider certain pre-requisites in addition:

- Create a subnet in Satellite
- Set the Network Address exactly same as Azure Subnet to be
selected
- While creating host, on Network Interfaces tab : 
    - Select the IPv4 Subnet created above
    - Enter the IPv4 address within the range of your Subnet